sources/packages/backend/apps/api/src/route-controllers/application/application.students.controller.ts

```diff
@@ -202,6 +205,17 @@ export class ApplicationStudentsController extends BaseController {
     ) {
       throw new UnprocessableEntityException("Invalid offering intensity.");
     }
+    if (payload.data.selectedProgram) {
+      const educationProgram =
+        await this.educationProgramService.getActiveEducationProgram(
+          payload.data.selectedProgram,
+        );
+      if (!educationProgram) {
+        throw new UnprocessableEntityException(
+          "Education Program is not active. Not able to create an application invalid request.",
+        );
+      }
+    }
```

Comment:
The offering validation similar to program is missing here on submit application.